articles/extensions/authorization-extension/v2/rules.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Using Rules with the Authorization Extension
+description: How to use information from the extension in rules
+---
+
+# Authorization Extension: Rules
+
+With [rules](/rules) that execute during the login process, you can grant them permissions to reach out to the Authorization Extension to do things like:
```

Comment:
You might want to rephrase this first sentence with sth simpler like Rules can be used with the Authorization Extension to do....